src/plugins/integrator.rs

```diff
@@ -310,4 +314,4 @@ fn clear_forces_and_impulses(mut forces: Query<ForceComponents, ForceComponentsC
             angular_ímpulse.clear();
         }
     }
-}
+}
```

Comment:
Files need to have a trailing newline
```suggestion
}

```